Lib/multiprocessing/popen_forkserver.py

```diff
@@ -43,8 +44,8 @@ def _launch(self, process_obj):
         buf = io.BytesIO()
         set_spawning_popen(self)
         try:
-            reduction.dump(prep_data, buf)
-            reduction.dump(process_obj, buf)
+            context.reduction.dump(prep_data, buf)
```

Comment:
Hmm... why doesn't this take a context object instead?

Same question for the spawn methods below.